server/profile.go

```diff
@@ -91,8 +91,87 @@ func GetProfile(w http.ResponseWriter, r *http.Request) {
 }
 
 func EditProfile(w http.ResponseWriter, r *http.Request) {
+	//Connect to datastore
+	ctx := context.Background()
+	dsClient, err := datastore.NewClient(ctx, Conf.Project)
```

Comment:
The initialization of the clients should be moved out of the serving path to avoid performance problems similar to the ones we had with the translation API. The pattern is usually to have these handlers as methods of an object which caches state.

Reply:
Could you explain to me a bit more how you would go about this?